lib/tbd/ua.rb

```diff
@@ -710,11 +710,13 @@ def ua_summary(date = Time.now, argh = {})
           case type
           when /rimjoist/i     then bloc[:pro][:rimjoists] += loss
           when /parapet/i      then bloc[:pro][:parapets ] += loss
+          when /roof/i         then bloc[:pro][:parapets ] += loss
           when /fenestration/i then bloc[:pro][:trim     ] += loss
           when /head/i         then bloc[:pro][:trim     ] += loss
           when /sill/i         then bloc[:pro][:trim     ] += loss
           when /jamb/i         then bloc[:pro][:trim     ] += loss
           when /corner/i       then bloc[:pro][:corners  ] += loss
+          when /balcony/i      then bloc[:pro][:balconies] += loss
```

Comment:
For generated UA summary tables, both _parapet_ and _non-parapet_ wall-to-roof intersections are lumped in a single category for brevity. Same for _balconysill_ vs _balcony_ types.